cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -185,51 +185,50 @@ func RunWrapper(s *options.CloudControllerManagerOptions, c *cloudcontrollerconf
 			}
 
 			panic("unreachable")
-		} else {
```

Comment:
How about not moving the `else` part out which should make the change clearer in this PR. Then do it separately. It is more reviewer friendly.
I'm in favor of the move here.

Reply:
it is reported by revive. The change in this pr will fix all the reported issues.

Reply:
Is there any way to split this PR since it is not reported before?

Reply:
the issues are ignored due to oom.

Reply:
Can you share an example.

Reply:
https://github.com/kubernetes-sigs/cloud-provider-azure/actions/runs/7019125850